config.yml

```
@@ -1413,6 +1413,19 @@ nodes:
 
           1.0i
           ^^^^
+  - name: ImplicitNode
+    fields:
+      - name: value
+        type: node
+    comment: |
+      Represents a node that is implicitly being added to the tree but doesn't
+      correspond directly to a node in the source.
+
+          { foo: }
+            ^^^^
```

This example seems not very clear, maybe:
```suggestion
          { foo: }
            ^^^^
gets translated to (assuming `foo` is a local variable):
HashNode([AssocNode(SymbolNode, ImplicitNode(LocalVariableReadNode))])
```